README.md

```diff
@@ -119,7 +119,7 @@ npm install
 cd ../
 git clone https://github.com/kubernetes-client/gen
 cd javascript
-../gen/openapi/typescript.sh src settings
+../gen/openapi/typescript.sh src/gen settings
```

Comment:
@silasbw does one need the `USERNAME` param?

`USERNAME=kubernetes ../gen/openapi/typescript.sh src/gen settings`

From https://github.com/kubernetes-client/javascript/pull/307

Reply:
I think so: that ensures we clone from the kubernetes org (and not, for example, someone's fork).